runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -80,6 +80,9 @@
     long mNativeContent;
     long mNativeWebContents;
 
+    // TODO(hengzhi.wu): This should be in a global context, not per XWalkView.
+    private double mDIPScale;
```

Comment:
please follow chromium naming rule, `DIP` -> `Dip`, please apply in all places in this PR.